core/git-access.c

```diff
@@ -843,18 +865,21 @@ static git_repository *create_local_repo(const char *localdir, const char *remot
 	opts.fetch_opts.callbacks.certificate_check = certificate_check_cb;
 
 	opts.checkout_branch = branch;
-	if (is_subsurface_cloud && !canReachCloudServer())
+	if (is_subsurface_cloud && !canReachCloudServer()) {
+		SSRF_INFO("git storage: cannot reach remote server");
 		return 0;
+	}
 	if (verbose > 1)
 		SSRF_INFO("git storage: calling git_clone()\n");
 	error = git_clone(&cloned_repo, remote, localdir, &opts);
 	if (verbose > 1)
```

Comment:
Shouldn't this be inside the next "if (error)" clause?

Reply:
I'm not sure I understand what you are saying.
Maybe I don't see what specific change you are commenting on. line 868-871 basically state "if this is the Subsurface cloud and we can't reach the cloud, then don't bother and return without an error, but at least log that we can't reach the cloud"

Or are you commenting on something else?

Reply:
I see the comment to be for line 875. Anyway, it should be for the following snippet that prints error message always when in verbose mode, even if no error occurred. Or if we want to print this message out even when there was no error, then we should not claim it to be error.
```
	if (verbose > 1)
		SSRF_INFO("git storage: returned from git_clone() with error %d\n", error);
	if (error) {
		SSRF_INFO("git storage: clone of %s failed", remote);
```
Of course, you did not touch this particular line, but it still jumped to my eye when skimming through this diff.

Reply:
right. got it. Let me contemplate how to improve the text :-)

Reply:
this should be more consistent now